exec/java-exec/src/test/java/org/apache/drill/test/rowSet/test/RowSetTest.java

```diff
@@ -309,13 +317,38 @@ private void testDoubleRW() {
     assertEquals(0, reader.column(0).getDouble(), 0.000001);
     assertTrue(reader.next());
     assertEquals(Double.MAX_VALUE, reader.column(0).getDouble(), 0.000001);
+    assertEquals(Double.MAX_VALUE, (double) reader.column(0).getObject(), 0.000001);
```

Comment:
Yes but my understanding is that `RowSetReader.column(0)` will return `ColumnReader` and you implemented `getObject` in `AbstractColumnReader` which is handing the type there. So `getObject` will return correct type of value ?